ISHPermissionKit/ISHPermissionsViewController.h

```diff
@@ -112,6 +112,7 @@ typedef void (^ISHPermissionsViewControllerCompletionBlock)(void);
 - (ISHPermissionRequestViewController *)permissionsViewController:(ISHPermissionsViewController *)vc requestViewControllerForCategory:(ISHPermissionCategory)category;
```

Comment:
No. The data source is required to provide a controller. There's even an assert in the code.